jaxrs-spec/src/main/asciidoc/chapters/appendix/_changes-since-1.1-release.adoc

```diff
@@ -14,7 +14,7 @@
 * <<status>>: Updated URLs to JSR pages, etc.
 * <<non_goals>>: Removed Client APIs as non-goal.
 * <<terminology>>: Added new terminology.
-* <<expert_group>>: List 2.0 expert group members.
+* <<project_team>>: List 2.0 expert group members.
```

Comment:
Why do the replace the term *here*? There never was a 2.0 project team, but only an expert group.

Reply:
I renamed the "Expert Group Members" chapter to "Project Team Members", which makes sense IMO, as we are talking about a project team from now on. And the changelog simply refers to this chapter, which I think is acceptable, even if the reference also has the new name.

Reply:
I know that. My question was a philosophical one, whether it could be incorrect to change *historical* chapters like this one. Actually at time of writing *that* chapter, the topics wasn't titled "Project Team", so it feels strange that it refers to the *new* name now. I think it would be more correct that it navigates to "Project Team" but still *says* "Expert Group".

Reply:
Got it. I agree that we could change the title of the link without changing the name of the section. However, this also feels a bit inconsistent.